hardhat.config.ts

```diff
@@ -130,7 +141,19 @@ const config: HardhatUserConfig = {
     runOnCompile: true
   },
   etherscan: {
-    apiKey: process.env.ETHERSCAN_API_KEY
+    apiKey: {
+        arbitrumGoerli: process.env.ETHERSCAN_API_KEY,
```

Comment:
Is there a default value for this? I don't want this to accidentally remove the ability to verify for non-goerli chains

Reply:
Ah, gotcha.. i have to look into this.

Reply:
Per our conversation in Slack, will leave this as it for this PR and address the issue when if others have problems verifying on other chains.

Reply:
Actually we can revert to the default API key. Doing that now.